documentation/man/features/mail.rst

```diff
@@ -20,15 +20,28 @@ It is common to deal with patch submissions to a mailing list, and
 **mail** functionality aims to wrap this tool to simplify its usage and
 integrate it with other **kw** functionalities.
 
+.. warning::
+  If any *<extra-args>* start with a ``-`` they **need** to be placed after the
+  ``--`` argument.
```

Comment:
The first time that I read it, I got a little bit confused because of this part:
`... [\--] [<rev-range>...] [-v<version>] [<extra-args>...]`
In some way, these `rev-range` and `-v<version>` are extra arguments, or did I miss something?

Reply:
Also, I expected that `-vX` could be placed before `--`.

Reply:
Yeah, `<rev-range>` and `-vX` work before and after the `--`, only `<extra-args>` that start with dashes **need** to be placed after the `--`. I will try to clarify this.